### PR TITLE
refactor: modify font awesome external link icon to material icon equivalent

### PR DESCRIPTION
**Pull request by hrk2023**
This pull request is in compliance to issue #3790 in the dart-lang/site-www repository.

### CODE REVIEW

File: src/_includes/sidenav-level-2.html [hunk 1]
```diff
@@ -20,7 +20,9 @@
     >{{entry2.title}}
     {%- if entry2.permalink == nil %} (TBC)
     {%- elsif entry2.permalink contains '://' -%}
-      <i class="fas fa-external-link-alt"></i>
+      <span class="material-icons md-24">
+        description
```
Comment (parlough) [inline src/_includes/sidenav-level-2.html:24]: I haven't had a chance to take a closer look at these changes yet, but why `description` here? I may be mistaken but I think the external icon we use elsewhere is called `launch`: https://fonts.google.com/icons?icon.query=launch

**Comment by hrk2023** [comment]
Sorry, my mistake. I had rectified the mistake. I have added [open_in_new ](https://fonts.google.com/icons?icon.query=open_in_new) material icon for external link.

**Comment by parlough** [comment]
Thanks for updating that!

\cc @domesticmouse or @johnpryan for review. I'll work with hrk to finalize the necessary downstream changes once this lands.

**Review by domesticmouse — APPROVED** [review]
Lgtm, but @johnpryan knows this area a lot better than I do

**Review by johnpryan — APPROVED** [review]

**Comment by domesticmouse** [comment]
@parlough are you ready for this to land?

**Comment by parlough** [comment]
@domesticmouse Yep, thanks! Don't update the submodule right away on the flutter/website side. It'll take some minor CSS changes I can handle.

**Comment by hrk2023** [comment]
Thanks for merging @domesticmouse, I will be submitting a PR with minor CSS addition in site-www repo for this change.

**Pull request merged**